swssconfig/sample/buffer.config.json

```diff
@@ -1,89 +1,72 @@
+        "BUFFER_POOL_TABLE:ingress_lossless_pool": {
+            "size": "3024486",
             "type": "ingress",
             "mode": "dynamic"
         },
         "OP": "SET"
     },
     {
-        "BUFFER_POOL_TABLE:i_pool1": {
-            "size": "3014688",
+        "BUFFER_POOL_TABLE:ingress_lossy_pool": {
+            "size": "6422528",
             "type": "ingress",
             "mode": "dynamic"
         },
         "OP": "SET"
     },
     {
-        "BUFFER_POOL_TABLE:e_pool0": {
-            "size": "8254464",
+        "BUFFER_POOL_TABLE:egress_lossless_pool": {
+            "size": "7291456",
             "type": "egress",
             "mode": "dynamic"
         },
         "OP": "SET"
     },
     {
-        "BUFFER_POOL_TABLE:e_pool1": {
-            "size": "7304064",
+        "BUFFER_POOL_TABLE:egress_lossy_pool": {
+            "size": "8254464",
             "type": "egress",
             "mode": "dynamic"
         },
         "OP": "SET"
     },
     {
-        "BUFFER_PROFILE_TABLE:i_port.profile0": {
-            "pool":"[BUFFER_POOL_TABLE:i_pool0]",
-            "xon":"0",
-            "xoff":"0",
+        "BUFFER_PROFILE_TABLE:ingress_lossless_profile": {
+            "pool":"[BUFFER_POOL_TABLE:ingress_lossless_pool]",
             "size":"0",
             "dynamic_th":"7"
         },
         "OP": "SET"
     },
     {
-        "BUFFER_PROFILE_TABLE:i_port.profile1": {
-            "pool":"[BUFFER_POOL_TABLE:i_pool1]",
-            "xon":"0",
-            "xoff":"0",
+        "BUFFER_PROFILE_TABLE:ingress_lossy_profile": {
+            "pool":"[BUFFER_POOL_TABLE:ingress_lossy_pool]",
             "size":"0",
             "dynamic_th":"7"
         },
         "OP": "SET"
     },
     {
-        "BUFFER_PROFILE_TABLE:e_port.profile0": {
-            "pool":"[BUFFER_POOL_TABLE:e_pool0]",
-            "xon":"0",
-            "xoff":"0",
-            "size":"4096",
+        "BUFFER_PROFILE_TABLE:egress_lossless_profile": {
+            "pool":"[BUFFER_POOL_TABLE:egress_lossless_pool]",
+            "size":"1518",
             "dynamic_th":"7"
         },
         "OP": "SET"
     },
     {
-        "BUFFER_PROFILE_TABLE:e_port.profile1": {
-            "pool":"[BUFFER_POOL_TABLE:e_pool1]",
-            "xon":"0",
-            "xoff":"0",
-            "size":"1126",
+        "BUFFER_PROFILE_TABLE:egress_lossy_profile": {
+            "pool":"[BUFFER_POOL_TABLE:egress_lossy_pool]",
+            "size":"4096",
             "dynamic_th":"7"
         },
         "OP": "SET"
     },
     {
-        "BUFFER_PROFILE_TABLE:i_port.pg01.profile": {
-            "pool":"[BUFFER_POOL_TABLE:i_pool0]",
-            "xon":"0",
-            "xoff":"0",
-            "size":"0",
-            "dynamic_th":"3"
-        },
-        "OP": "SET"
-    },
-    {
-        "BUFFER_PROFILE_TABLE:i_port.pg34.profile": {
-            "pool":"[BUFFER_POOL_TABLE:i_pool1]",
+        "BUFFER_PROFILE_TABLE:pg_lossless_profile": {
```

Comment:
Xoff / Xon still seems to be in old units of cells (96 bytes) and not in new units of bytes
In addition, Xoff still seems to be from beginning of buffer and not from end
expected xon is 35686 (371*96)
expected xoff is 31744 (67430-371*96)